core/src/main/java/org/ethereum/beacon/core/spec/ChainSpec.java

```diff
@@ -43,7 +43,7 @@ public UInt64 getBeaconChainShardNumber() {
         }
 
         @Override
-        public UInt64 getMaxCasperVotes() {
+        public int getMaxCasperVotes() {
```

Comment:
Why? Should we?

Reply:
This constant is applied to a list/array size which have `int` type in Java. This is what stays behind this change.

Reply:
We may need to use any other constant could some day in Java code.

Reply:
I'm going to review all those casts and types at some point

Reply:
This particular constant limits a size of `SlashableVoteData.custodyBit0Indices + SlashableVoteData.custodyBit1Indices`. And not gonna be used at some other place.